src/Rapid/Client.php

```diff
@@ -444,6 +444,8 @@ private function doQueryInvoiceReference($invoiceReference)
      */
     private function doCreateCustomer($apiMethod, $customer)
     {
+        $paymentInstrument = $customer['PaymentInstrument'] ?? null;
```

Comment:
This line of code assumes that $customer is an array. However, it [can be](https://github.com/eWAYPayment/eway-rapid-php/blob/master/src/Rapid/Contract/Client.php#L222) a `Customer` or an array.

Implementing this version upgrade causing issues for us because we are passing a `Customer` object - `Cannot use object of type Eway\Rapid\Model\Customer as array`.